attest/attest-tool/attest-tool.go

```diff
@@ -63,10 +63,10 @@ func runCommand(tpm *attest.TPM) error {
 		fmt.Printf("VendorInfo: %x\n", info.VendorInfo)
 		fmt.Printf("Manufactorer: %v\n", info.Manufacturer)
 
-	case "make-aik":
-		k, err := tpm.NewAIK(nil)
+	case "make-ak":
```

Comment:
Command line flags, being untyped, are notorious for breaking things in wierd and wonderful ways. Could we do both to avoid breaking existing users?

```go
case "make-ak", "make-aik":
  // ...
```